proxy/src/main/java/org/dragonet/proxy/network/translator/ItemBlockTranslator.java

```diff
@@ -26,35 +26,65 @@
 	// vars
 	public static final int UNSUPPORTED_BLOCK_ID = 165;
 	public static final String DRAGONET_COMPOUND = "DragonetNBT";
-	public static final Map<Integer, Integer> PC_TO_PE_OVERRIDE = new HashMap<>();
-	public static final Map<Integer, Integer> PE_TO_PC_OVERRIDE = new HashMap<>();
+	public static final Map<Integer, ItemEntry> PC_TO_PE_OVERRIDE = new HashMap<>();
+	public static final Map<Integer, ItemEntry> PE_TO_PC_OVERRIDE = new HashMap<>();
 	public static final Map<Integer, String> NAME_OVERRIDES = new HashMap<>();
 
 	static {
 		swap(125, 157); // Double Slab <-> Activator Rail
-		onewayOverride(126, 158); // Slab <-> NULL
-		onewayOverride(95, 20, "Stained Glass"); // Stained Glass = Glass
-		onewayOverride(160, 102, "Stained Glass Pane"); // Stained Glass Pane = Glass Pane
-		onewayOverride(119, 90); // End portal -> Nether portal
-		onewayOverride(176, 63, "Banner"); // Sign =\_
-		onewayOverride(177, 68, "Banner"); // Wall sign =/ We send banner as sign [Banner]
-		onewayOverride(36, 248);
-		onewayOverride(84, 248);
-		onewayOverride(122, 248);
-		onewayOverride(130, 248);
-		onewayOverride(137, 248);
-		onewayOverride(138, 248);
-		onewayOverride(160, 248);
-		onewayOverride(166, 248);
-		onewayOverride(168, 248);
-		onewayOverride(169, 248);
-		onewayOverride(176, 248);
-		onewayOverride(177, 248);
-		onewayOverride(188, 248);
-		onewayOverride(189, 248);
-		onewayOverride(190, 248);
-		onewayOverride(191, 248);
-		onewayOverride(192, 248);
+		toPEOverride(126, 158); // Slab <-> NULL
+		toPEOverride(95, 241); // Stained Glass
+		toPEOverride(36, 248);
+		toPEOverride(166, 95); //barrier -> invisible bedrock
+		toPEOverride(188, new ItemEntry(85, 1));
+		toPEOverride(189, new ItemEntry(85, 2));
+		toPEOverride(190, new ItemEntry(85, 3));
+		toPEOverride(191, new ItemEntry(85, 5));
+		toPEOverride(192, new ItemEntry(85, 4));
+		toPEOverride(203, 202); //purpur stairs
```

Comment:
add also :
- (157, 28); //detector_rail
- (166, 95); //barrier -> invisiblebedrock
- (168, 248); //prismarine -> update_block
- (169, 248); //prismarine_bricks -> update_block
- (198, 208); //end_rod
- (199, 240); //chorus_plant
- (202, 201); //purpur_pillar -> purpur_block
- (208, 198); //grass_path
- (210, 188); //repeated command block
- (211, 189); //chained command block

Reply:
I think we don't have to add update blocks. Minecraft automatically replaces unknown blocks with update block